ethminer/main.cpp

```diff
@@ -320,6 +320,8 @@ class MinerCLI
 
         app.add_flag("--cl-nobin", m_CLSettings.noBinary, "");
 
+        app.add_flag("--cl-noexit", m_CLSettings.noExit, "");
```

Comment:
I know there is `--cl-nobin` but can we remove the negation from the option? Make it `--cl-fast-exit`.

Reply:
> Can't really use --cl-fastexit if I want fast exit to be the default. The cmd. line parser allows flags to be turned on, but not turned off!